example/30-worker.yaml

```
@@ -87,5 +87,9 @@ spec:
     # workerconfig is used to specify the interface of gcp local ssd
     #dataVolumes:
     #  - type: SCRATCH
+    # serviceAccount:
+    #   email: foo@bar.com
+    #   scopes:
+    #   - https://www.googleapis.com/auth/cloud-platform
```

Hm, actually, it should be
```suggestion
   # dataVolumes:
   #   - type: SCRATCH
   # providerConfig:
   #   apiVersion: gcp.provider.extensions.gardener.cloud/v1alpha1
   #   kind: WorkerConfig
   #   serviceAccount:
   #     email: foo@bar.com
   #     scopes:
   #     - https://www.googleapis.com/auth/cloud-platform
```